rust/run.js

```diff
@@ -115,7 +115,7 @@ async function main() {
 
 async function testInteraction(interaction) {
   console.log("\n\nExecuting interaction:", interaction);
-  const executionResult = await smartweaveContract.handle(interaction);
+  const executionResult = await smartweaveContract.handle_entrypoint(interaction);
```

Comment:
this will require a change in SDK (note to myself ;-))

Reply:
No need anymore. I renamed hi function back to `handle`